testsuite/tests/ui/master/test_tenant.py

```diff
@@ -0,0 +1,107 @@
+"""
+    Testing ground for testing master on tenant creation / edit / deletion ...
```

Comment:
I don't understand this doc

Reply:
an edit to this doc was made, to make it a bit more clear.